packages/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/utils/DependencyUtils.kt

```diff
@@ -84,9 +84,18 @@ internal object DependencyUtils {
     }
   }
 
+  // Allows react-native-tvos to get its dependencies correctly with its custom Maven group name
+  fun readGroupString(propertiesFile: File): String {
+    val reactAndroidProperties = Properties()
+    propertiesFile.inputStream().use { reactAndroidProperties.load(it) }
+    return reactAndroidProperties["GROUP"] as? String ?: defaultGroupString
+  }
+
   fun Project.mavenRepoFromUrl(url: String): MavenArtifactRepository =
       project.repositories.maven { it.url = URI.create(url) }
 
   fun Project.mavenRepoFromURI(uri: URI): MavenArtifactRepository =
       project.repositories.maven { it.url = uri }
+
+  val defaultGroupString = "com.facebook.react"
```

Comment:
Let's move this out of this class to the top level and make it a 
```suggestion
  internal const val DEFAULT_GROUP_STRING = "com.facebook.react"
```

Reply:
Done